admin_manual/issues/index.rst

```diff
@@ -48,8 +48,18 @@ helpful. See:
   bug in older SabreDAV versions and information for mod_security problems)
 * `0 byte files <http://sabre.io/dav/0bytes>`_ (Reasons for empty files on the 
   server)
-* `Clients <http://sabre.io/dav/clients/>`_ (A comprehensive lists of WebDAV 
+* `Clients <http://sabre.io/dav/clients/>`_ (A comprehensive list of WebDAV 
   clients, and possible problems with each one)
 * `Finder, OS X's built-in WebDAV client 
   <http://sabre.io/dav/clients/finder/>`_ 
   (Describes problems with Finder on various webservers)
+
+General Troubleshooting
+-----------------------
+
+.. note:: The data directory on the server is exclusive to ownCloud and must not be modified manually.
+
+Disregard this can lead to unwanted behaviours like:
```

Comment:
@wakeup 
Thanks :+1: 

Also added another note what to use instead of the direct access to the data directory.